lib/src/chain.dart

```diff
@@ -70,7 +70,7 @@ class Chain implements StackTrace {
   /// considered unhandled.
   ///
   /// If [callback] returns a value, it will be returned by [capture] as well.
-  static /*=T*/ capture/*<T>*/(/*=T*/ callback(),
+  static/*=T*/ capture/*<T>*/(/*=T*/ callback(),
```

Comment:
I'm not sure if this syntax works. You might need an explicit `dynamic` in the return type:

```dart
static dynamic/*=T*/ capture/*<T>*/(/*=T*/ callback(),
```

(Or you could use the real generic method syntax, but that means bumping up the minimum version constraint on the SDK.)